packages/components-organizations-registrar/package.json

```diff
@@ -70,7 +70,7 @@
     "build": "vite build",
     "build:watch": "vite build --watch",
     "prepublish": "vite build",
-    "test": "jest --coverage",
+    "test": "NODE_OPTOINS=--experimental-vm-modules jest --config ./jest.config.cjs --runInBand --coverage",
```

Comment:
I know it's not new, but an opportunity to complain 😅
Setting env vars like that doesn't work on windows
I've been having issues with that since it was added 😔